lib/puppet/file_bucket/dipper.rb

```diff
@@ -109,6 +109,21 @@ def restore(file, sum)
     end
   end
 
+  # List Filebucket content.
+  def list(fromdate, todate)
+    source_path = "#{@rest_path}#{@checksum_type}/"
+    fromdate ||= '0:0:0 1-1-1970'
```

Comment:
Same comment as above.  Can we just pass nil through here for `fromdate` and `todate`?